doc/en/components/radio.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Radio and Radio Group
+_description: With $ProductName$ Radio Button and Radio Group controls, developers can seamlessly present lists of options for users to select for better UI in template-driven and reactive forms.
+_keywords: $ProductName$, UI controls, $Platform$ widgets, web widgets, UI widgets, $Platform$, Native $Platform$ Components Suite, Native $Platform$ Controls, Native $Platform$ Components Library, $Platform$ Radio Button components, $Platform$ Radio Button controls, $Platform$ Radio Group component, $Platform$ Radio Group control
+---
+
+# $Platform$ Radio & Radio Group
+
+<p class="highlight">The $ProductName$ Radio Button component allows the user to select a single option from an available set of options that are listed side by side.</p>
+
+## $ProductName$ Radio Example
+
+<code-view style="height: 205px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/inputs/radio-group" alt="$Platform$ Radio & Radio Group Example">
+</code-view>
+
+<div class="divider--half"></div>
+
+### Usage
+
+To get started with the radio web component, first you need to install the $ProductName$ by typing the following command:
+
+```cmd
+npm install igniteui-webcomponents
+```
+
+The next step is to import the IgcRadioComponent and IgcRadioGroupComponent in the typescript file where you want to use them:
+
+```ts
+import { IgcRadioComponent } from "igniteui-webcomponents/src/components/radio";
+import { IgcRadioGroupComponent } from "igniteui-webcomponents/src/components/radio-group";
+```
+
+The simplest way to start using the radio is as follows:
+
+```html
+<igc-radio-group>
+  <igc-radio>Apple</igc-radio>
+  <igc-radio>Banana</igc-radio>
+  <igc-radio>Mango</igc-radio>
+  <igc-radio>Orange</igc-radio>
+</igc-radio-group>
+```
+
+> [!WARNING]
+> The radio component doesn't work with the standart `<form>` element. Use `<igc-form>` instead.
+
+## Examples
+
+### Label
+
+To provide a meaningful label for the radio, simply place some text between the opening and closing tags:
+
+```html
+<igc-radio>Apple</igc-radio>
+```
+
+You can specify if the label should be positioned before or after the switch toggle by setting the `label-position` attribute of the switch. Allowed values are `before` and `after`(default):
```

Comment:
Switch mentioned instead of radio!
```suggestion
You can specify if the label should be positioned before or after the radio button by setting the `label-position` attribute. Allowed values are `before` and `after`(default):
```

Reply:
Add API links